# EDA

## Trainラベルの確認

In [22]:
import pandas as pd

# データ読み込み
df = pd.read_csv("../data/train.csv")
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 4238 entries, 0 to 4237
Data columns (total 2 columns):
 #   Column         Non-Null Count  Dtype 
---  ------         --------------  ----- 
 0   image          4238 non-null   object
 1   gender_status  4238 non-null   int64 
dtypes: int64(1), object(1)
memory usage: 66.3+ KB


In [21]:
df["gender_status"].value_counts(sort=True)

1    1294
0    1210
4     834
2     375
3     371
7     100
6      53
5       1
Name: gender_status, dtype: int64

### 教師ラベル/予測ラベル（性別×身分）

1 : 男-武士 = 1294  
0 : 男-貴族 = 1218   
4 : 女-貴族 = 834  
2 : 男-化身 = 375  
3 : 男-庶民 = 371  
7 : 女-庶民 = 100  
6 : 女-化身 = 53  
5 : 女-武士 =  1  
